src/js/Nostr.ts

```diff
@@ -87,10 +88,18 @@ const DEFAULT_RELAYS = [
   'wss://brb.io',
 ];
 
+const SEARCH_RELAYS = [
```

Comment:
why r u hard code this? doesn't this client have settings for it

Reply:
Afaik only @brugeman's relay supports keyword search atm

Reply:
Discussed this w/ @mmalmi 

Only my relay supports keyword search atm. The NIP is proposed, but it won't be considered until some clients implement this. We could tell users that they need to add my relay to their list for search to work, but that means a) it won't really work for anyone, and b) they're forced to save my relay in their contact list and promote it essentially. 

So an approach to hard-code the relay for a while just for search seems better, user's relay lists won't be affected, all users will be able to use the search right away, if some more relays start supporting it we can extend the search relay list, and if NIP is accepted and all relays start supporting it then we drop the hard-coded list entirely.

Reply:
In the interest of privacy and decentralization in that case, this feature should probably live behind a feature flag in settings until NIP gets approved